lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -56,8 +55,16 @@ def test_savefig_to_stringio(format, use_log, rcParams, orientation,
         if not mpl.rcParams["text.usetex"]:
             title += " \N{MINUS SIGN}\N{EURO SIGN}"
         ax.set_title(title)
-        fig.savefig(s_buf, format=format, orientation=orientation)
-        fig.savefig(b_buf, format=format, orientation=orientation)
+        allowable_exns = []
```

Comment:
```suggestion
        allowable_exceptions = []
```
Took me a minute to understand what this is.

Reply:
sure, edited